src/main/java/Duke.java

```diff
@@ -1,10 +1,177 @@
+import java.util.Locale;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public static final String LINE_PRINT = "____________________________________________________________\n";
```

Comment:
However, this name sounds a bit weird to me, maybe can just change to 'LINE' or something?